src/main/java/org/sonarlint/intellij/promotion/PromotionProvider.kt

```diff
@@ -188,7 +200,21 @@ class PromotionProvider(private val project: Project) {
         }?.let { org.sonarsource.sonarlint.core.client.utils.Language.fromDto(it) }
     }
 
+    private fun findExtraLanguages(languagesToPromote: Set<Language>, languages: Set<Language>): List<String> {
+        val extraLanguagesToPromote = ArrayList<String>()
+
+        languagesToPromote.forEach { language ->
+            languages.find {
+                it.name == language.name
+            }?.let { extraLanguagesToPromote.add(org.sonarsource.sonarlint.core.client.utils.Language.fromDto(it).label) }
+        }
+
+        return extraLanguagesToPromote
+    }
+
     private fun showPromotion(notifications: SonarLintProjectNotifications, content: String) {
+        PropertiesComponent.getInstance().setValue(LAST_PROMOTION_NOTIFICATION_DATE, Instant.now().toEpochMilli().toString())
```

Comment:
Is this change needed? It's all done after in the `if()`, I think there's an issue

Reply:
It was leftover code while testing forgot to delete